### DISCUSSION TITLE
How to lodge GPUI related bugs/features

### DISCUSSION THREAD

**Discussion by Quadriphobs1** [zed-industries/zed#8473] [Feature Requests]
How does the team plan to address creating issues relating to GPUI as an external contributor/user?

**Comment by mrnugget** [comment]
I guess I'm missing something, but I think the idea is that you just create an issue in this issue tracker here. Or did you mean something else?

**Comment by Quadriphobs1** [comment]
This repo is predominantly for the `zed` app, there might be some issue that might be specifically related to improving GPUI that might not tie in with the immediate roadmap of zed especially if the idea is to make GPUI usable outside of zed

**Reply by mrnugget** [reply]
I think for now that gpui is in the repo it's fine to add the issues here and just give them the gpui tag.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
